website/docs/flink-quick-start-guide.md

```diff
@@ -401,6 +401,82 @@ select * from hudi_table/*+ OPTIONS('read.streaming.enabled'='true')*/;
 This will give all changes that happened after the `read.start-commit` commit. The unique thing about this
 feature is that it now lets you author streaming pipelines on streaming or batch data source.
 
+## Non-Blocking Concurrency Control
```

Comment:
Refer to the comment above on moving this section to other Flink related docs. We can call this out in the https://hudi.apache.org/docs/next/flink-quick-start-guide#where-to-go-from-here section.

Reply:
done.. moved to flink-sql-writer section in writeing_data page.